.github/meson_actions/action.yml

```diff
@@ -3,19 +3,34 @@ description: "checkout repo, build, and test numpy"
 runs:
   using: composite
   steps:
-  - name: Install dependencies
-    shell: bash
-    run: pip install -r build_requirements.txt
   - name: Build
     shell: 'script -q -e -c "bash --noprofile --norc -eo pipefail {0}"'
     env:
       TERM: xterm-256color
-    run:
-      spin build -- ${MESON_ARGS[@]}
+    run: |
+      echo "::group::Installing Build Dependencies"
+      pip install -r build_requirements.txt
+      echo "::endgroup::"
+      echo "::group::Building NumPy"
+      spin build --clean -- ${MESON_ARGS[@]}
+      echo "::endgroup::"
+
+  - name: Meson Log
+    shell: bash
+    if: always()
+    run: |
+      echo "::group::Meson Log"
+      cat build/meson-logs/meson-log.txt
+      echo "::endgroup::"
```

Comment:
This step being included with all the others results in log browsing being a bit slow, since it's >10,000 lines as a result of `meson-log.txt` being so large. It's hidden by default though, and I think this is fine as a price to pay for such a concise `linux_simd.yml` file. So I wanted to note this, but no change needed.